transforms/universal/filter/python/src/filter_transform.py

```diff
@@ -67,6 +67,10 @@ def __init__(self, config: dict):
         self.logical_operator = config.get(filter_logical_operator_key, filter_logical_operator_default)
         self.columns_to_drop = config.get(filter_columns_to_drop_key, filter_columns_to_drop_default)
 
+        # Temporarily here to test if this can allow use to process files that are required to be read by polars for mm
```

Comment:
consider removing the 4 commented lines or restore file to its original commit.

Reply:
done